### PR TITLE
Better exploit searching

### PR DESCRIPTION
**Pull request by linted**
**TL;DR**
- Removed abandoned dependency
- Multi-threading search of exploit db
- No longer needs to download exploitdb, and supports custom path to the database

This branch removes the abandoned dependency we were using to parse exploitdb. The major reason for moving away from that dependency is due to the change in structure of exploitdb's archive. The new parsing method uses kali's built in searchsploit database, and also supports custom csv files locations. Since we are doing the parsing ourself now, the multithreading library speeds up the process by running 10 threads at once in a pool doing the searches. In the future we can add a `-j` option to configure the number of jobs, but that is for another pull request.

### CODE REVIEW

**Comment by linted** [comment]
This pull request should solve #12

**Review by Whurbin — APPROVED** [review]

**Comment by Whurbin** [comment]
Looks good
